src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,83 @@
+package edu.kis.vh.nursery;
+
+/**
+ * The type Default counting out rhymer.
+ */
+public class DefaultCountingOutRhymer {
+
+    /**
+     * The constant INT.
+     */
+    public static final int INT = 12;
+    /**
+     * The constant INT1.
+     */
+    public static final int INT1 = -1;
+    /**
+     * The constant INT2.
+     */
+    public static final int INT2 = 11;
```

Comment:
This constant is related to INT (bad name). Changing only one leads to an error (inconsistency).